app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -370,8 +370,6 @@ ShortFormApplicationController = (
   $scope.checkAfterLiveInTheNeighborhood = (preference) ->
     # preference is either neighborhoodResidence or antiDisplacement
     if ShortFormApplicationService.applicationHasPreference(preference)
-      # Neighborhood provides automatic liveInSf preference
-      ShortFormApplicationService.copyNeighborhoodToLiveInSf(preference)
```

Comment:
@chrisdolendo yeah I agree we can get rid of this happening in `ShortFormApplicationController` but I'm saying we should also add a callback on the member select dropdown, basically the same behavior as above: `if @preference == 'neighborhoodResidence' || @preference == 'antiDisplacement' ... copyNeighborhood...`